epregressions/tests/diffs/test_math_diff.py

```diff
@@ -45,6 +45,19 @@ def test_small_diff_in_watts_files(self):
         self.assertEqual(0, response[2])  # big diffs
         self.assertEqual(2, response[3])  # small diffs
 
+    def test_bad_numeric_raises_exception(self):
+        """This tests the ability to handle bad numerics which cause weird errors in MathDiff"""
+        with self.assertRaises(KeyError):
```

Comment:
Test that math_diff does raise an exception with bad numerics.  I don't really think there's much else we could do better than just throwing the exception and printing a nice result, so this is fine.